beta-src/src/components/wd-popover/WDPopover.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import { useRef } from "react";
+import { Popover, Box } from "@mui/material";
+
+interface WDPopoverProps {
+  children: React.ReactNode;
+  handleClose: (event: React.MouseEvent<HTMLButtonElement>) => void; // A function that sets isOpen to false
+  isOpen: boolean;
+  trigger: React.ReactNode; // The popover trigger; must have an onClick callback attached that sets isOpen to true
```

Comment:
same as above on comment styling.

let's rename this prop to `popoverElement` with a description of:

"A component that opens or closes the Popover when clicked"